high-availability.helmfile.yaml

```diff
@@ -115,7 +144,7 @@ releases:
         value: true
       - name: logger.level
         value: debug
-      - name: notationCert
+      - name: notationCerts[0]
```

Comment:
Hi @akashsinghal , I want to double check here did we remove support for notationCert in our charts? do you think we need to update the releaseNotes to include this as a breaking change?

Reply:
notationCert is not removed but is marked as deprecated. I will add a note on deprecation in release notes.

Reply:
Discussed offline that we need to consider adding tests to make sure these deprecated values still are supported